Interpreter/Program.cs

```diff
@@ -6,19 +6,12 @@
 while (true)
 {
     var input = Console.ReadLine();
-    var lexer = new Lexer.Lexer(Logger);
-    var result = lexer.MakeTokens(input, "test");
-
-    if (result.tokens == null && result.error != null)
-    {
-        Console.WriteLine(result.error.ToString());
-        return;
-    }
+    var tokens = Lexer.Lexer.Instance(Logger).MakeTokens(input, "test");
```

Comment:
Lexer.Lexer.Instance isn't nice to read....this is caused by the fact that nameSpace and class has the same name....can you fix that?